test/eth.subscribe.ganache.js

```diff
@@ -63,6 +63,20 @@ describe('subscription connect/reconnect', function () {
         assert.equal(0, web3.eth._requestManager.subscriptions.size);
     });
 
+    it('unsubscribes given an id', function (done) {
+        assert.equal(0, web3.eth._requestManager.subscriptions.size);
+        subscription = web3.eth
+            .subscribe('newBlockHeaders')
+            .on('connected', function (result) {
+                assert(result)
+                assert.equal(1, web3.eth._requestManager.subscriptions.size);
+                subscription.unsubscribeById(subscription.id); // Stop listening..
```

Comment:
I put this comment here to reference what I'm talking about

Instead of doing:

```javascript
subscription = web3.eth.subscribe(...')
subscription.unsubscribeById(...)
```

It would be cool if you could find a way to do

```javascript
Subscription.unsubscribeById(...)
```

Or maybe it would be

```javascript
web3.eth.subscribe.unsubscribeById(...)
```

so you don't have to have access to an instance of `Subscription`. But maybe what you have already works like that too? Idk, can you tell me? :)

Reply:
can probably make use of looking through `web3.eth._requestManager.subscriptions` and loop through the `id`s! Great suggestion

Reply:
Cool! the request manager was helpful :). 
It can now be called with
```
web3.eth. removeSubscriptionById( id)
```